modules/ocf_kubernetes/manifests/master/loadbalancer/ssl.pp

```diff
@@ -0,0 +1,20 @@
+class ocf_kubernetes::master::loadbalancer::ssl(
+  $server_name,
+) {
+
+  $cnames = ldap_attr_multi($server_name, 'dnsCname')
+
+  $cnames.each |String $alias| {
+    $service_fqdn = "${alias}.ocf.berkeley.edu"
+    ocf::ssl::bundle { $service_fqdn:
+      domains => [$service_fqdn, "${alias}.ocf.io"],
+    }
+  }
+
+  file { '/etc/ssl/ocf-certs.txt':
+    mode    => '0400',
+    owner   => 'haproxy',
```

Comment:
These won't exist until haproxy is provisioned

Reply:
Can we make this dependent on the haproxy package?

Reply:
yes